lib/ansible/utils/vars.py

```diff
@@ -265,8 +263,6 @@ def _isidentifier_PY3(ident):
 portable between the two. The following changes were made:
 
     * disallow non-ascii characters (Python 3 allows for them as opposed to Python 2)
-    * True, False and None are reserved keywords (these are reserved keywords
```

Comment:
I think we can just rename `_isidentifier_PY3` to `isidentifier` and move this doc string there at this point.

Reply:
"rename" wasn't precise. `isidentifier` already exists and is a public interface. We just assigned `_isidentifier_PY3` or `_isidentifier_PY2` to it depending on the Python version. Now that only `_isidentifier_PY3` remains it is no longer needed.